core/babe/babe.go

```diff
@@ -0,0 +1,71 @@
+package babe
```

Comment:
I think this could be placed in its own package `consensus` where we will have `babe` and `grandpa`. Thoughts?

Reply:
yeah that makes sense, I'll change it